server/README.md

```diff
@@ -142,6 +142,65 @@ You can erase the API data fixtures created by `make run-api` with:
 make clean
 ```
 
+## Running the API server locally with real data
+
+You can also run the api server and all the other services with real data using docker compose. For this setup you'll need the following:
+- docker compose,
+- a service account with the corresponding json credentials file to connect to the CloudSQL database,
+- the credentials for the CloudSQL databases.
+
+Spin up the container with postgres and the container with cloud sql proxy:
+```
+$ docker-compose up cloud_sql_proxy postgres
+```
+
+This step requires the credentials of the service account as a json file. The default path is `./credentials.json`, otherwise it can be set using the `CLOUD_SQL_PROXY_CREDENTIALS_FILE` env var.
```

Comment:
```suggestion
This step requires the credentials of the service account in a json file. The default path is `./credentials.json`, otherwise it can be set using the `CLOUD_SQL_PROXY_CREDENTIALS_FILE` env var.
```